nemo/collections/nlp/callbacks/joint_intent_slot_callback.py

```diff
@@ -96,7 +96,43 @@ def eval_epochs_done_callback(global_vars, graph_fold):
     logging.info("Sampled s_preds: [%s]" % list2str(slot_preds[i : i + sample_size]))
     logging.info("Sampled slots: [%s]" % list2str(slot_labels[i : i + sample_size]))
 
-    plot_confusion_matrix(intent_labels, intent_preds, graph_fold)
+    # remove labels from label_ids that don't appear in the dev set
```

Comment:
How generic is that plot?

Reply:
in the sense: it would be great to use it with any kinds of prediction-target pairs ;)